crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -49,29 +41,30 @@ impl AwsConverter {
     }
 }
 
-#[async_trait]
+pub fn make_valid_topic_or_panic(topic_name: &str) -> Topic {
+    Topic::new(topic_name).expect("Invalid topic name")
+}
+
 impl Converter for AwsConverter {
     type Error = ConversionError;
+    type Input = MqttMessage;
+    type Output = MqttMessage;
 
-    fn get_mapper_config(&self) -> &MapperConfig {
-        &self.mapper_config
-    }
-
-    async fn try_convert(&mut self, input: &Message) -> Result<Vec<Message>, Self::Error> {
+    fn convert(&mut self, input: &Self::Input) -> Result<Vec<Self::Output>, Self::Error> {
         let default_timestamp = self.add_timestamp.then(|| self.clock.now());
 
         // serialize with ThinEdgeJson for measurements, for alarms and events just add the timestamp
         let payload = if input.topic.name.starts_with("tedge/measurements") {
             let mut serializer = ThinEdgeJsonSerializer::new_with_timestamp(default_timestamp);
-            tedge_api::parser::parse_str(input.payload_str()?, &mut serializer)?;
+            tedge_api::parser::parse_str(input.payload_str().unwrap(), &mut serializer)?;
 
             serializer.into_string()?
         } else if input.topic.name.starts_with("tedge/events")
             || input.topic.name.starts_with("tedge/alarms")
             || input.topic.name.starts_with("tedge/health")
```

Comment:
The converter need not worry about `tedge/health` messages anymore.

Reply:
I agree, need to check with Marcel, what was the intention to put them there.

Reply:
This is not to respond to health *checks* but to forward health *status* to the cloud. Please change nothing here.